typescript/looker/rtl/nodeTransport.ts

```diff
@@ -172,23 +174,26 @@ export class NodeTransport implements ITransport {
 }
 
 async function parseResponse(contentType: string, res: Response) {
-  if (contentType.match(/application\/json/g)) {
-    try {
-      // return await res.json()
-      const result = await res.body
-      return result instanceof Object ? result : JSON.parse(result)
-    } catch (error) {
-      console.log(res.body)
-      return Promise.reject(error)
+  const mode = responseMode(contentType)
+  let result = await res.body
+  if (mode === ResponseMode.string) {
+    if (!isUtf8(contentType)) {
+      // always convert to UTF-8 from whatever it was
+      result = Buffer.from(result.toString(), 'utf8')
```

Comment:
I may be mis-reading the `Buffer` docs but it seems like `Buffer.toString()` defaults to output using the utf8 encoding so is this necessary?

Reply:
You **may** be right, but I couldn't get it to behave the way I expected without this. I definitely want to improve this code if possible.